rbac/management/principal/view.py

```diff
@@ -123,11 +122,12 @@ def get(self, request):
         options["principal_type"] = principal_type
 
         # Get either service accounts or user principals, depending on what the user specified.
-        if principal_type == SA_KEY:
-            resp = self.service_accounts_from_it_service(request, user, query_params, options)
-        else:
+        if principal_type == USER_KEY:
```

Comment:
```suggestion
        if principal_type in ("", USER_KEY):
```
I'm getting 500 when trying this request: `"http://$RBAC_LOCAL/api/rbac/v1/principals/?type="`

Reply:
this case fixed here https://github.com/RedHatInsights/insights-rbac/pull/1534